scapy/sendrecv.py

```diff
@@ -115,44 +115,43 @@ def _get_pkt():
                 raise _BreakException()
 
     try:
-        try:
```

Comment:
I understand why you removed this try, but this PR itself is already tricky, and it makes it real hard to review + is not needed.
If it’s not bothering you too much, it would be better that you leave the current useless try.

All of those fixes will surely be fixed through the PEPin project, which will come in time

Reply:
Done